src/web/app/pages-help/instructor-help-page/instructor-help-students-section/instructor-help-students-section.component.html

```diff
@@ -0,0 +1,225 @@
+<div>
+  <h2 class="color-blue" *ngIf="searched_terms == -1">Students</h2>
+  <!-- Section -->
+  <h3 *ngIf="searched_terms == -1">Student Records</h3>
+  <!-- Question -->
+  <div class="instr-help-qn0" *ngIf="show_arr[0]">
+    <p>
+      <button type="button" class="btn btn-outline-primary" (click)="isCollapsed = !isCollapsed" [attr.aria-expanded]="!isCollapsed" aria-controls="answer">
+      How do I view a student's profile?
+      </button>
+    </p>
+    <div id="answer" [ngbCollapse]="!isCollapsed">
```

Comment:
the `instr-help-qn` class is used for the script but is the `answer` id necessary?

Reply:
nope it is not needed. Removed the id